packages/core-bridge/src/lib.rs

```diff
@@ -243,6 +250,32 @@ fn start_bridge_loop(
                                     .await;
                                 break;
                             }
+                            Request::PollLogs { callback } => {
+                                let logs = core.fetch_buffered_logs();
+                                if !logs.is_empty() {
+                                    send_result(event_queue.clone(), callback, |cx| {
+                                        let logarr = cx.empty_array();
+                                        for (i, cl) in logs.into_iter().enumerate() {
+                                            let logobj = cx.empty_object();
+                                            let level = cx.string(cl.level.to_string());
+                                            logobj.set(cx, "level", level).unwrap();
+                                            // eww, time conversion. Use neon date if possible in napi-5
+                                            let ts = cx.number(cl.millis_since_epoch() as f64);
```

Comment:
TBH I think it might be more convenient to work with the timestamps as numbers.
Dates are a bit of a pain in JS but that's what [temporal](https://tc39.es/proposal-temporal/docs/) is for.

Reply:
Also, it should be safe to use napi-5

Reply:
I'll just keep it as is then